scripts/us_epa/parent_company/pre_process.py

```diff
@@ -0,0 +1,299 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A script to parse and pre-process the Parent Company Info for Facilities tracked by EPA."""
```

Comment:
Is this essentially doing company resolution?

Reply:
Yes it is!

Reply:
Done!